install-steps.md

```diff
@@ -1,193 +1,821 @@
+# Extract the baremetal installer
+./oc adm release extract --registry-config "${pullsecret_file}" --command=$cmd --to "${extract_dir}" ${RELEASE_IMAGE}
+~~~
+
+## Configure the install-config and run the installer
+
+1. Configure the `install-config.yaml` (Make sure you change the pullSecret and sshKey)
+    ~~~yaml
+    apiVersion: v1
+    baseDomain: <domain>
+    metadata:
+      name: <cluster-name>
+    networking:
+      machineCIDR: <public-cidr>
+    compute:
+    - name: worker
+      replicas: 2
+    controlPlane:
+      name: master
+      replicas: 3
+      platform:
+        baremetal: {}
+    platform:
+      baremetal:
+        apiVIP: <api-ip>
+        ingressVIP: <wildcard-ip>
+        dnsVIP: <dns-ip>
+        provisioningBridge: provisioning
+        externalBridge: baremetal
+        hosts:
+          - name: openshift-master-0
+            role: master
+            bmc:
+              address: ipmi://<out-of-band-ip>
+              username: <user>
+              password: <password>
+            bootMACAddress: <NIC1-mac-address>
+            hardwareProfile: default
+          - name: openshift-master-1
+            role: master
+            bmc:
+              address: ipmi://<out-of-band-ip>
+              username: <user>
+              password: <password>
+            bootMACAddress: <NIC1-mac-address>
+            hardwareProfile: default
+          - name: openshift-master-2
+            role: master
+            bmc:
+              address: ipmi://<out-of-band-ip>
+              username: <user>
+              password: <password>
+            bootMACAddress: <NIC1-mac-address
+            hardwareProfile: default
+          - name: openshift-worker-0
+            role: worker
+            bmc:
+              address: ipmi://<out-of-band-ip>
+              username: <user>
+              password: <password>
+            bootMACAddress: <NIC1-mac-address
+            hardwareProfile: default
+          - name: openshift-worker-1
+            role: worker
+            bmc:
+              address: ipmi://<out-of-band-ip>
+              username: <user>
+              password: <password>
+            bootMACAddress: <NIC1-mac-address
+            hardwareProfile: default
+    pullSecret: '<pull_secret>'
+    sshKey: '<ssh_pub_key>'
+    ~~~
+
+NOTE: Ensure to change the appropriate variables to match your environment.
+
+2. Create a directory to store cluster configs
+    ~~~sh
+    mkdir ~/clusterconfigs
+    cp install-config.yaml ~/clusterconfigs
+    ~~~
+
+3. Ensure all baremetal nodes are powered off prior to installing the OpenShift cluster
+   ~~~sh
+   ipmitool -I lanplus -U <user> -P <password> -H <management-server-ip> -p 6230 power off
+   ~~~
+
+4. Run the OpenShift Installer
+    ~~~sh
+    ./openshift-baremetal-install --dir ~/clusterconfigs --log-level debug create cluster
+    ~~~
```

Comment:
If deploying masters + workers it will eventually timeout... in that case, the `wait-for install-complete` will be required if you want to see a successful finish.

Reply:
@e-minguez I was able to get a successful deployment without the need for `wait-for install-complete` Spoke to @schmaustech and he has also gotten installations to work without the need of this.

Reply:
Going to resolve this as it was not needed.

Reply:
I think it would be nice to included as optional or a just in case... I've hit the timeout all the times I've deployed in baremetal :man_shrugging: